spark8t/literals.py

```diff
@@ -0,0 +1,3 @@
+MANAGED_BY_LABELNAME = "app.kubernetes.io/managed-by"
+PRIMARY_LABELNAME = "app.kubernetes.io/spark8t-primary"
+SPARK8S_LABEL = "spark8t"
```

Comment:
This seems more of internal configuration to me than a python module.
Why don't we use something like `configparse` instead?